src/k8s/pkg/k8s/client/cluster.go

```diff
@@ -46,6 +46,7 @@ func (c *Client) Bootstrap(ctx context.Context, bootstrapConfig apiv1.BootstrapC
 	}
 	if err := c.m.NewCluster(hostname, addrPort, config, time.Second*30); err != nil {
 		// TODO(neoaggelos): print message that bootstrap failed, and that we are cleaning up
+		fmt.Println("Cleaning up, error was", err)
```

Comment:
nit: `fmt.Println("Cleaning up, error was: ", err)` makes it slightly more readable.